serialosc/osc.md

```diff
@@ -8,59 +8,59 @@ permalink: /serialosc/osc/
 
 # osc : opensound control / serialosc protocol
 
-what is serialosc? how does it work?
+What is serialosc? How does it work?
 
 ## discovering and connecting to serialosc devices
 
-serialosc server listens on port 12002.
+The serialosc server listens on port 12002.
 
-when devices are connected, serialosc spawns new ports for each device. querying the server allows you to discover the port number for each device. (this supersedes the zeroconf method, which is still in place for legacy compatibility).
+When devices are connected, serialosc spawns new ports for each device. querying the server allows you to discover the port number for each device. For those who remember it, this supersedes the zeroconf method, which is still in place for legacy compatibility.
```

Comment:
we can remove the zeroconf reminder, since that method as primary was only in place for a very narrow time period over a decade ago